CONVERT_SERVICE/convert_service/convert_image/scripts/convert_image.py

```diff
@@ -0,0 +1,134 @@
+#
+# @python_parameters.py Copyright (c) 2021 Jalasoft.
+# 2643 Av Melchor Perez de Olguin, Colquiri Sud, Cochabamba, Bolivia.
+# All rights reserved.
+#
+# This software is the confidential and proprietary information of
+# Jalasoft, ("Confidential Information").  You shall not
+# disclose such Confidential Information and shall use it only in
+# accordance with the terms of the license agreement you entered into
+# with Jalasoft.
+#
+
+from wand.image import Image
+from wand.display import display
+from wand.color import Color
+
+
+class Param:
+    def __init__(self, filename, grayscale, blur, adaptive_sharpen, resize, flip, flop, rotate, noise, charcoal, matrix,
+                 implode, vignette):
+        self.filename = filename
+        self.grayscale = grayscale
+        self.blur = blur
+        self.adaptive_sharpen = adaptive_sharpen
+        self.resize = resize
+        self.flip = flip
+        self.flop = flop
+        self.rotate = rotate
+        self.noise = noise
+        self.charcoal = charcoal
+        self.matrix = matrix
+        self.implode = implode
+        self.vignette = vignette
+
+    def get_filename(self):
+        return self.filename
+
+    def get_grayscale(self):
+        return self.grayscale
+
+    def get_blur(self):
+        return self.blur
+
+    def get_adaptive_sharpen(self):
+        return self.adaptive_sharpen
+
+    def get_resize(self):
+        return self.resize
+
+    def get_flip(self):
+        return self.flip
+
+    def get_flop(self):
+        return self.flop
+
+    def get_rotate(self):
+        return self.rotate
+
+    def get_noise(self):
+        return self.noise
+
+    def get_charcoal(self):
+        return self.charcoal
+
+    def get_matrix(self):
+        return self.matrix
+
+    def get_implode(self):
+        return self.implode
+
+    def get_vignette(self):
+        return self.vignette
+
+
+class ConvertImage:
+
+    def convert(self, param):
```

Comment:
In this function no consistency in blank lines. Some "if" statements are surrounded by blank lines and others don't.
@jpsandovaln what is the correct convention about this? I read this but I am not sure.
https://stackoverflow.com/questions/47105228/should-if-statements-be-surrounded-by-blank-lines-to-follow-pep8

Reply:
we need to define it as team, 
if we consider use blank lines to surrounded if statement
in all part of code we need to apply it.